openstack/compute/v2/extensions/volumeattach/requests.go

```diff
@@ -0,0 +1,109 @@
+package volumeattach
+
+import (
+	"errors"
+
+	"github.com/racker/perigee"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/openstack/compute/v2/servers"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// CreateOptsExt adds a VolumeAttachment option to the base CreateOpts.
+type CreateOptsExt struct {
+	servers.CreateOptsBuilder
```

Comment:
`CreateOptsExt` is used to add extension parameters to the base `CreateOpts` from the `servers` package. After looking at the OpenStack Nova source code,  I couldn't find where a volume could be attached to a server at creation time. Were you able to get this to work for you?